libexec/bats-core/bats-format-tap-stream

```diff
@@ -92,18 +92,18 @@ buffer_with_truncation() {
   shift
   local string
 
-  printf -v 'string' -- "$@"
+  printf -v 'string' "%s" "$@"
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2059

Not sure if this is going to break anything, tbh. Need someone to validate.